tests/goodMapFile.map

```diff
@@ -39,3 +39,4 @@ LARGE_BAR.NUMBER                  0x00000001    0x00000000    0x00000004    0x10
 MODULE0.INTERRUPT_VOID1               0x00000000    0x00000000    0x00000000    0x00000000       0         0        0 INTERRUPT3 
 MODULE0.INTERRUPT_VOID2               0             0             0             0                0         0        0 INTERRUPT2 
 MODULE0.INTERRUPT_TYPE                0x00000001    0x00000068    0x00000004    0x00000001       18        5        0 INTERRUPT6  
+MODULE1.TEST_AREA2                    0x00000001    0x00000070    0x00000008    0x00000001       32        0        0
```

Comment:
This is not an area. "area" was the old name of 1D registers, or memory blocks with multiple scalars that for technical reasons are somehow connected (e.g. memory region mapped from a sub device into the PCIe address space).

This register is a 64 bit scalar. Also. don't put it into the interrupt section. It should be higher up in the file.